test/errors-adjacency-matrix-modifications.js

```diff
@@ -35,13 +35,13 @@ describe("errors-adjacency-matrix-modifications", function () {
         });
     });
 
-
-    describe("extra-column-end-of-sheet", function () {
-        it("should return warnings count error", function () {
-            test.warningsCountError("test-files/adjacency-matrix-modifications/extra-column-end-of-sheet-input.xlsx", 49062);
-            test.warningsCountError("test-files/adjacency-matrix-modifications/extra-column-end-of-sheet-output.xlsx", 49062);
-        });
-    });
+    // ISSUE #491
+    // describe("extra-column-end-of-sheet", function () {
+    //     it("should return warnings count error", function () {
+    //         test.warningsCountError("test-files/adjacency-matrix-modifications/extra-column-end-of-sheet-input.xlsx", 49062);
+    //         test.warningsCountError("test-files/adjacency-matrix-modifications/extra-column-end-of-sheet-output.xlsx", 49062);
+    //     });
+    // });
```

Comment:
OK, so this is one of the deferred failing tests. I’m OK with leaving this commented out if they are to be addressed immediately after.